common/components/controllers/AboutUsController.jsx

```diff
@@ -24,49 +32,67 @@ class AboutUsController extends React.PureComponent<{||}, State> {
   constructor(): void {
     super();
     this.state = {
-      aboutUs: null,
+      project: null,
       projectId: parseInt(window.DLAB_PROJECT_ID),
       showBiographyModal: false,
       modalPerson: null,
-      personTitle: null,
-    }
-    this.handleShow = this.handleShow.bind(this);
+    };
+    this.handleShowBio = this.handleShowBio.bind(this);
     this.handleClose = this.handleClose.bind(this);
   }
 //componentDidMount and loadProjectDetails copied from AboutProjectController, since we're retrieving a project's information the same way
```

Comment:
Can probably remove this comment, since we're no longer loading project details the same way